src/main/java/at/hannibal2/skyhanni/config/features/inventory/chocolatefactory/ChocolateFactoryRabbitWarningConfig.java

```diff
@@ -34,6 +35,16 @@ public class ChocolateFactoryRabbitWarningConfig {
     @ConfigEditorSlider(minValue = 1, maxValue = 20, minStep = 1)
     public int repeatSound = 20;
 
+    @Expose
+    @ConfigOption(name = "Flash Screen", desc = "Flash the screen in the given color when a special rabbit appear.")
+    @ConfigEditorBoolean
+    public boolean flashScreen = false;
```

Comment:
maybe also an option to show for non special rabbits, normal stray rabbits? (also this would be easier to test the flash haha)